dask/array/core.py

```diff
@@ -3049,7 +3049,7 @@ def broadcast_to(x, shape, chunks=None):
 
     See Also
     --------
-    numpy.broadcast_to
+    :function:`numpy.broadcast_to`
```

Comment:
Do you know if `:function:` is the same as `:func`?

Reply:
Also, is including these the `:function:` role necessary to get the linkn? I thought that numpydoc handles these, though I may be wrong.

Reply:
`:func:`  works, any other options doesn't  generate a link